guides/plugins/plugins/storefront/add-icons.md

```diff
@@ -0,0 +1,71 @@
+# Add custom icons
+
+## Overview
+
+In order to design a outstanding and customized User Experience, it's important to think about icons, ideally 
+self-designed. To support you in that, Shopware provides the possibility to use custom icons in the storefront.
+
+## Prerequisites
+
+In order to follow this guide easily, you first need to have a functioning plugin installed. Head over to our 
+[plugin base guide](../plugin-base-guide.md) to create a plugin, if you don't know how it's done yet.
+Also knowing and understanding SCSS will be quite mandatory to fully understand what's going on here.
+Furthermore, it might be helpful to read the guide on how to [handle own assets](./add-custom-assets.md) in your 
+plugin before you start with this one.
+
+## Adding icon
+
+In order to add any icons to the Storefront, you use our `sw_icon` twig action. This way, an icon of choice is 
```

Comment:
Maybe we can also explain where to put the icon. Something like:

First put your icon inside your custom Theme. The default path for icons is:

```
<YourTheme>/src/Resources/app/storefront/dist/assets/icon/default
```

You can also provide "solid" icons or any other custom pack names which can be configured later with the "pack" parameter. You can do that by creating a folder with the pack name. By default shopware looks inside the "default" folder.

```
<YourTheme>/src/Resources/app/storefront/dist/assets/icon/<pack-name>
```